.tests/testScript_plotting.m

```diff
@@ -24,7 +24,7 @@
 line(gca,'XData',a*u.acre,'YData',sqrt(a*u.acre),'LineWidth',7)
 xlabel('area')
 ylabel('length')
-assert(axishasunits('acre','ft'))
+assert(axishasunits('acre','ft'))%FIXME: sqrt(a*u.acre) results in m instead of ft
```

Comment:
Because the class is cleared followed by setting the preferred displayUnits to ft for lengths (line 9), `sqrt(u.acre)` displays and plots as ft (and this passes for me).